docs/reference/synonyms/apis/put-synonyms-set.asciidoc

```diff
@@ -7,6 +7,10 @@
 
 Creates or updates a synonyms set.
 
+NOTE: Synonyms sets are limited to a maximum of 10,000 synonym rules per set.
```

Comment:
Should we say: "Currently synonyms sets..."

as we plan to address this in the future.

Reply:
As this is going to be backported to previous versions, I'd say no? Maybe we can change this with the version in which it is fixed when we do. Is that something we do in docs?

Reply:
I agree that it's best not to add "currently," partly for the backporting reasons and also because sometimes plans change :) 

>Maybe we can change this with the version in which it is fixed when we do.

+1 to that 👍